Mage.Sets/src/mage/cards/d/DeepGnomeTerramancer.java

```diff
@@ -0,0 +1,119 @@
+package mage.cards.d;
+
+import java.util.UUID;
+
+// Abilities
+import mage.abilities.effects.common.search.SearchLibraryPutInPlayEffect;
+import mage.abilities.keyword.FlashAbility;
+import mage.abilities.TriggeredAbilityImpl;
+
+// Cards
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+
+// Constants
+import mage.constants.CardType;
+import mage.constants.Outcome;
+import mage.constants.SubType;
+import mage.constants.Zone;
+
+// Filter
+import mage.filter.FilterCard;
+
+// Game
+import mage.game.events.GameEvent;
+import mage.game.Game;
+import mage.game.permanent.Permanent;
+
+// Base
+import mage.MageInt;
+
+// Target
+import mage.target.common.TargetCardInLibrary;
+
+// Watchers
+import mage.watchers.common.PlayLandWatcher;
+
+/**
+ * @author rgudmundsson
+ */
+public final class DeepGnomeTerramancer extends CardImpl {
+
+    public DeepGnomeTerramancer(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[] { CardType.CREATURE }, "{1}{W}");
+
+        this.subtype.add(SubType.GNOME);
+        this.subtype.add(SubType.WIZARD);
+
+        this.power = new MageInt(2);
+        this.toughness = new MageInt(2);
+
+        // Flash
+        this.addAbility(FlashAbility.getInstance());
+
+        // Whenever one or more lands enter the battlefield under an opponent's control
+        // without being played, you may search your library for a Plains card, put it
+        // onto the battlefield tapped, then shuffle. Do this only once each turn.
+        this.addAbility(new DeepGnomeTerramancerTriggeredAbility().setDoOnlyOnce(true));
```

Comment:
+1, just checked out this pull request and it functions correctly with line 57 changed to:
`this.addAbility(new DeepGnomeTerramancerTriggeredAbility().setDoOnlyOnce(true), new PlayLandWatcher());`
Before adding the watcher, it would NPE on a land being played.